### PR TITLE
Update MainService.Plugins.cs

### PR DESCRIPTION
**Pull request by chenzhitong**
Text alignment
https://github.com/neo-project/neo-modules/pull/453#discussion_r557100757

### CODE REVIEW

**Comment by erikzhang** [comment]
I don't think it can align the texts.

**Comment by chenzhitong** [comment]
before
image

after
image

**Comment by chenzhitong** [comment]
> I don't think it can align the texts.

Now it can.

File: neo-cli/CLI/MainService.Plugins.cs [hunk 1]
```diff
@@ -107,7 +107,7 @@ private void OnPluginsCommand()
                 foreach (Plugin plugin in Plugin.Plugins)
                 {
                     if (plugin is Logger) continue;
-                    Console.WriteLine("\t" + plugin.Name + "\t" + plugin.Description);
+                    Console.WriteLine(string.Format("\t{0,-10}\t{1}", plugin.Name, plugin.Description));
```
Comment (erikzhang) [inline neo-cli/CLI/MainService.Plugins.cs:110]: `ApplicationLogs` has 15 characters.

Comment (chenzhitong) [reply neo-cli/CLI/MainService.Plugins.cs:110]: image

Comment (erikzhang) [reply neo-cli/CLI/MainService.Plugins.cs:110]: Maybe `{0,-15}` is better?

Comment (chenzhitong) [reply neo-cli/CLI/MainService.Plugins.cs]: {0,-15}
image

{0,-16}
image

Comment (erikzhang) [reply neo-cli/CLI/MainService.Plugins.cs]: 15 is better.

Comment (chenzhitong) [reply neo-cli/CLI/MainService.Plugins.cs]: I revoved \t and set Name.maxLength = 20

**Comment by erikzhang** [comment]
Why revert?

**Comment by chenzhitong** [comment]
> Why revert?
.NET Core Test / Test (ubuntu-latest) (pull_request)  faild.  No Pass

Maybe this way of writing will not compile successfully on Ubuntu

**Review by erikzhang — APPROVED** [review]

**Pull request merged**